src/dhali/transaction_utils.py

```diff
@@ -87,6 +94,66 @@ def _transactional_validation(
 
     return to_claim
 
+
+def _validation(
+    db,
+    root_private_payment_claim_doc_ref,
+    ledger_client,
+    parsed_claim,
+    single_request_cost_estimate: int,
+    settle_delay,
+) -> float:
+    
+    root_private_payment_claim_doc = root_private_payment_claim_doc_ref.get()
+
+    updating_payment_claim = True
+    if root_private_payment_claim_doc.exists and not root_private_payment_claim_doc.to_dict():
```

Comment:
Is the to_dict check implicitly checking something? Or is it just to see if we can parse it as a dict?

Reply:
Discussed in person, will add comment maybe